data/_types/lists/index.json

```diff
@@ -0,0 +1,167 @@
+{
+  "layout": {
+    "template": "layout"
+  },
+  "meta": {
+    "title": "Lists - Types - UK Parliament",
```

Comment:
Is this title correct?

Reply:
As these *.json files are for local development, I have decided to use meta.title in conjunction with our lighthouse audit. So in essence, yes meta.title are correct.

See ticket: WEBDEV-1060

Reply:
Got you, we wanted to reflect the path of the file `./_types/lists`, however we can change this if required. Note it is also the same for `./_types/things` and `./_types/utilities`